### PR TITLE
Update for 2.1 with new bulk action for association

### PR DESCRIPTION
**Pull request by Delphineray**

### CODE REVIEW

File: content/md/products-super-power/product-mass-actions.md [hunk 1]
```diff
@@ -142,6 +144,35 @@ This operation can be only performed for non variant products, so products not a
...
+6.  Click on the `Add associations` button
+  ![Add association](../img/Products_BulkAddAssociation.png)
+7.  The picker to select products or product models opens
+8.  Filter on the categories, on the label or identifier or add filters to find the products or product models to associate
+9.  Tick in the grid the products or product models to associate, they are displayed in the basket
+  ![Add association](../img/Products_BulkPickerAssociation.png)
+10.  Click on the `Confirm` button
+11.  Confirm the bulk action with the `Confirm` button located in the middle of the screen
+  ![Add association](../img/Products_BulkConfirmAssociation.png)
+12.  The products selected will be associated to the products and product models/
```
Comment (KmilleF) [inline content/md/products-super-power/product-mass-actions.md]: typo at the end on the sentence.

File: content/md/products-super-power/product-mass-actions.md [hunk 2]
```diff
@@ -142,6 +144,35 @@ This operation can be only performed for non variant products, so products not a
...
+
+:::warning
+[Product models](/articles/what-about-products-variants.html#what-is-a-product-model) have no association, if you select product models in the grid and mass associate it to products or product models, all their children variant products will be associated.
+:::
+
+:::info
+You can view the products associations in the product form, in the `Associations` tab, refer to [this article](/articles/products-associations.html) for more details.
```
Comment (KmilleF) [inline content/md/products-super-power/product-mass-actions.md]: Product associations like product grid? Product Form?

File: content/md/products-super-power/product-mass-actions.md (hunk 1 above)
Comment (Delphineray) [reply content/md/products-super-power/product-mass-actions.md]: oh yes

File: content/md/products-super-power/product-mass-actions.md (hunk 2 above)
Comment (Delphineray) [reply content/md/products-super-power/product-mass-actions.md]: changed

**Pull request merged**